app/models/person.rb

```diff
@@ -180,7 +180,7 @@ def full_last_name
   # Return [String, nil]
   #   convenience, maybe a delegate: candidate
   def orcid
-    identifiers.where(type: 'Identifier::Global::Orcid').first&.cached
```

Comment:
@gdower is there a difference?  Cached should == identifier I think?  Think of cached as the getter, it's safer in the long run to point to it.  If there is a problem with it currently we should re-factor how it is populated.

Reply:
I generated a Person for my account to test the orcid and cached wasn't set on save, although maybe I'm supposed to manually populate it? Should I also change the User model to point orcid and wikidata_id at cached instead of identifier?

https://github.com/SpeciesFileGroup/taxonworks/blob/development/app/models/user.rb#L462